src/content/docs/custom-pipeline/setup.mdx

```diff
@@ -1,209 +1,186 @@
 ---
 title: Setting up your computer
-description: Setting up your computer for creating a custom pipeline.
+description: Setting up your computer to create a custom pipeline.
 ---
 
-## Prototype pipeline development container
+import CloneIcon from '~icons/codicon/source-control'
+import RemoteIcon from '~icons/codicon/remote'
+import ExtensionsIcon from '~icons/codicon/extensions'
+import CoffeeIcon from '~icons/codicon/coffee'
+import WaitIcon from '~icons/ic/round-access-time'
 
-## Requirements
+## Quick setup : VSCode and **devcontainers**
 
-- [VS Code](https://code.visualstudio.com) &geq; 1.95
-- [Docker](https://www.docker.com/get-started/) &geq; 24 (we recommend using [Docker Desktop](https://www.docker.com/products/docker-desktop))
+### Prerequisites
 
-### Configure Docker access
-
-The simplest way of installing everything Docker is to use [Docker Desktop](https://www.docker.com/products/docker-desktop).
-You can also go the [engine way](https://docs.docker.com/engine/install) and install Docker manually.
-
-Once installed, you need to configure some access rights to the Docker daemon. The easiest way to do this is to add your user
-to the `docker` group. This can be done with the following command :
+The fastest way to start coding is by installing [VSCode](https://code.visualstudio.com) (&geq; 1.95)
```

Comment:
More emphasis on what has to be installed before continuing. It may not be in words but maybe with an extra paragraph to delineate everything.